src/main/java/io/jenkins/plugins/intotorecorder/transport/GrafeasTransport.java

```diff
@@ -29,19 +29,19 @@ public static class GrafeasInTotoMetadata {
         // This class exists to represent the signed document format for Grafeas
         // in-toto links.
 
-        @edu.umd.cs.findbugs.annotations.SuppressWarnings("URF_UNREAD_FIELD")
+        @edu.umd.cs.findbugs.annotations.SuppressFBWarnings("URF_UNREAD_FIELD")
         private ArrayList<Signature> signatures = new ArrayList<Signature>();
-        @edu.umd.cs.findbugs.annotations.SuppressWarnings("URF_UNREAD_FIELD")
+        @edu.umd.cs.findbugs.annotations.SuppressFBWarnings("URF_UNREAD_FIELD")
```

Comment:
Can we open a ticket to handle this and other unread fields? We should assess if those fields are needed / can be refactored out etc.